zwave_js_server/model/statistics.py

```diff
@@ -40,13 +40,14 @@ class RouteStatistics:
     client: "Client"
     data: RouteStatisticsDataType
     protocol_data_rate: ProtocolDataRate = field(init=False)
-    repeaters: list["Node"] = field(init=False)
+    repeaters: list["Node"] = field(init=False, default_factory=list)
     route_failed_between: tuple["Node", "Node"] | None = field(init=False, default=None)
 
     def __post_init__(self) -> None:
         """Post initialize."""
         self.protocol_data_rate = ProtocolDataRate(self.data["protocolDataRate"])
-        assert self.client.driver
+        if not self.client.driver:
+            return
```

Comment:
This is not good, because it means the object will contain incorrect data. 

Your idea of a function is better, as it is calculated. You can also do `@property` and then you don't need code changes.

You can also turn it into a `@cached_property` if you know it's never going to change, that way it's only calculated once during lifetime of the object.